live/src/fix_bootconfig

```diff
@@ -13,12 +13,18 @@
 #    This script (a) adjusts the boot config, (b) runs xorriso, and (c) makes
 #    it zIPL bootable on s390x.
 #
+set -x
 
-# KIWI config
-test -f .profile && . .profile
 
 dst=${1#iso:}
 
+# KIWI config
+# Copy it to $dst from where it can be later read by xorriso
+if [ -f .profile ]; then
+    source .profile
+    cp .profile "$dst"
+fi
+
 arch=`uname -m`
 
 # removing live root setting from command line - it's in /etc/cmdline.d instead
```

Comment:
I think this is useless now since we have "static" bootconfig (with expansion of $kiwi_iname)